src/nvim/decoration.h

```diff
@@ -23,9 +23,12 @@ typedef enum {
   kVTOverlay,
   kVTWinCol,
   kVTRightAlign,
+  kVTInline,
 } VirtTextPos;
 
-EXTERN const char *const virt_text_pos_str[] INIT(= { "eol", "overlay", "win_col", "right_align" });
+EXTERN const char *const virt_text_pos_str[] INIT(=
```

Comment:
no line break between `=` and `{`

Reply:
Hmm, but this is the formatting applied by uncrustify, otherwise the linter will complain

Reply:
Are you sure it was the right version of uncrustify with our config file?

Reply:
```
$ uncrustify --version                                                       
Uncrustify-0.76.0
~~
while the config file says `Uncrustify-0.76.0_f` - is the `_f` suffix important?

But the CI also seemed to have the same issue: https://github.com/neovim/neovim/actions/runs/4855242666/jobs/8653579277#step:14:156